automatic/minikube/update.ps1

```diff
@@ -0,0 +1,46 @@
+[CmdletBinding()]
+param([switch] $Force)
+
+Import-Module AU
+
+$domain   = 'https://github.com'
+$releases = "$domain/kubernetes/minikube/releases/latest"
+
+function global:au_BeforeUpdate {
+  Get-RemoteFiles -Purge -NoSuffix
```

Comment:
add the following parameter to this call:
`-FileNameBase minikube`

Then you don't need to rename the executable below.